src/wre/game.ts

```diff
@@ -31,16 +31,19 @@ const game: Natives = {
 
     *getObjectsIn(self: RuntimeObject, position: RuntimeObject): Execution<RuntimeValue> {
       const visuals = self.get('visuals')!
-
       const result: RuntimeObject[] = []
 
       for(const visual of visuals.innerCollection!) {
         const otherPosition = yield* this.send('position', visual)
-        const samePosition = yield* this.send('onSameCell', self, position, otherPosition!)
-        if(samePosition!.innerBoolean)
+        const samePosition = (
+          position.get("x")?.innerValue && otherPosition?.get("x")?.innerValue &&
+          position.get("x")?.innerValue == otherPosition?.get("x")?.innerValue &&
+          position.get("y")?.innerValue && otherPosition?.get("y")?.innerValue &&
+          position.get("y")?.innerValue == otherPosition?.get("y")?.innerValue
```

Comment:
podrías laburarlo directamente así

```suggestion
        const x = position.get("x")?.innerValue
        const otherX = otherPosition.get("x")?.innerValue
        const y = position.get("y")?.innerValue
        const otherY = otherPosition.get("y")?.innerValue
        
        const samePosition = (
          x && otherX &&
          x == otherX &&
          y && otherY &&
          y && otherY
```

Reply:
Incluso podrías preguntar primero si x e y tienen valores, para no preguntar por el valor de otherPosition

Reply:
listo (creo), me costó tanto descubrir como sacar el valor de `x` e `y` que cuando anduvo lo dejé así nomas...